package.json

```diff
@@ -54,12 +55,14 @@
     "@vitejs/plugin-react": "^3.0.0",
     "babel-jest": "^29.2.2",
     "cross-env": "^7.0.3",
+    "cypress": "^12.2.0",
     "deepdash": "^5.3.9",
     "desm": "^1.3.0",
     "eslint": "^8.26.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-prettier": "^8.5.0",
     "eslint-config-react-app": "^7.0.1",
+    "eslint-plugin-cypress": "^2.12.1",
```

Comment:
if we install the eslint plugin, we should add it to the eslint config, otherwise it will do nothing